FE/src/components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  Content,
+  Nickname,
+  PersonalInfoLeftDiv,
+  PersonalInfoRightDiv,
+  PointMarket,
+  ProfileImage,
+  Rank,
+  Section,
+  Topic,
+} from './ProfileInfo.styles';
+import Programmer from '/images/Programmer.png';
+
+const dummyTopic = ['등수', '포인트', '전체질문', '전체답변', '채택답변'];
+const dummyContent = ['1817', '543', '13건', '6건', '3건'];
+
+export function ProfileInfo() {
```

Comment:
export 방식의 통일성을 지켜주세요!
```suggestion
export default function ProfileInfo() {
```